crates/nu-explore/src/nu_common/lscolor.rs

```diff
@@ -0,0 +1,115 @@
+use std::fs::symlink_metadata;
+
+use lscolors::LsColors;
+use nu_ansi_term::{Color, Style};
+use nu_engine::env_to_string;
+use nu_protocol::engine::{EngineState, Stack};
+use nu_utils::get_ls_colors;
+
+use super::NuText;
+
```

Comment:
I'd add a comment here explaining what LS_COLORS actually is (a terminal env-var standard for colourising `ls` output that predates Nushell - maybe with links to places that explain it more thoroughly).